data/wanderer/wanderer news.txt

```diff
@@ -0,0 +1,127 @@
+			"Wanderer civilian"
+			"Wanderer"
+	message
+		word
+			"Long ago, we [rebirthed, cleansed] a group of worlds in the south. I wonder if any [aliens, sentient beings] live there now?"
+			"It is a shame we must [fight, defend against] the Hai. I wish we had the opportunity to [meet, socialize with] their peaceful brethren."
+			"Alien societies [fascinate, interest] me. I [hope, desire] for more opportunities to meet with them."
+	name
+		word
+			"Wanderer linguist"
+			"Linguist"
+	message
+		word
+			"The Hai language is [complicated, convoluted], but learning it has yielded much [insight, knoweldge] about the Hai and [aliens, non-Wanderers] in general."
+			"It is difficult to [pronounce, speak] Hai words, because their [physiology, structure] is so much different than ours." 
+			
+news "Wanderer civilians, dialog"
+	to show
+		has "Wanderers Conversation: offered"
+	location
+		government "Wanderer"
+		not attributes "evacuation"
+	name
+		word
+			"Wanderer"
+			"Wanderer worker"
+			"Wanderer traveller"
+	message
+		word
+			"Whenever I go on a [journey, trip] through space I always bring some of my [favorite, most cherished] potted plants. It gets [lonely, barren] when you are far from nature." 
+			"I have [heard of, known of] a human captain in Wanderer space, but seeing you is still quite [strange, odd]."
+			"I cannot [imagine, fathom] being unable to fly. How do you manage to get anywhere?"
+			"I wonder what kinds of [creatures, beings] live on your home planet."
+			"With that jump drive, you must be able to [see, witness] all kinds of wonderous [creatures, forms of life] from across the galaxy. I wish I had such access."
+news "Wanderer scientists"
+	to show
+		has "Wanderers Conversation: offered"
+	location
+		government "Wanderer"
+		not attributes "evacuation"
+	name
+		word
+			"Wanderer scientist"
+			"Scientist"
+			"Wanderer ecologist"
+			"Ecologist"
+	message
+		word
+			"I recently [travelled, journeyed] to another world on a scientific [mission, study], where we discovered a new species of plant. How [exciting, thrilling]!"
+			"The best part of my [job, task] is the thrill of [discovery, finding new knoweldge]."
+news "Wanderer soldier"
+	to show
+	has "Wanderers Conversation: offered"
+	location
+		government "Wanderer"
+		attributes "evacuation"
+	name
+		word
+			"Wanderer soldier"
+			"Soldier"
+			"Wanderer conscript"
+			"Conscript"
+	message
+		word
+			"Society does not [like, appreciate] what I do, but it is [necessary, essential] in times like these."
+			"I wish I did not have to [fight, do battle] for a living. It is a [savage, unelegant] way to get by."
+			"I joined the [defense fleet, navy] because I wanted to protect lifeforms. Instead I am [battling, killing] them. It brings me great [sadness, sorrow] to see such [destruction, waste]."
+			"I should've [become, studied to be] an ecologist like my parents [wanted, desired] me to."
+			"I kill one so that [others, several] may be [saved, spared]."
```

Comment:
I'm not honestly sure that any of this is in keeping with the Wanderer's as they've been described thus far.

Reply:
Could you elaborate?

Reply:
The conversations in the plot imply that the wanderers simply did not engage in combat until extremely recently. The idea that any of them are now soldiers without having had a different career prior to that is just incongruous.
Also if this only fires on the evacuation planets it's a very narrow window where they're able to show up.

Reply:
On Varu K'est, the description after the Unfettered invade implies that the Wanderers did have a standing army before the invasion: `Before the Unfettered invaded, the only Wanderer settlements remaining here were the military bases they had set up to defend their frontier.` I could change this to have it only spawn on that world.

Reply:
How feasible is it to have this only appear on that world until after the Unfettered take the first Wanderer worlds?

Reply:
I'm very new to ES syntax, but perhaps I could use a combo of `government: Wanderer` and `attributes: wanderer military`?

Reply:
@MasterOfGrey are you still opposed to these lines? I just want your opinion before I do anything with them.

Reply:
On Varu K'est it has the description you mentioned, but very early in Wanderers Iktat Rek says directly to your face, `"At first only a single Hai ship [visited, invaded] us, but back then we had no defenses in place to stop it."`

Implying that they didn't actually have those bases before the unfettered invaded, as the description suggests.

So It comes down to whether you're in favour of the narrative canon or the written world base. 

I tend to lean on the narrative canon: which would suggest that none of these lines should be used, and the world description also needs to change.

On the other hand, you could justify altering the narrative canon to fit the world base instead, in which case these are fine, but I feel like that could introduce more issues with why it took the Wanderer's so long to dig up and get back into rebuilding their military ships.

Reply:
I think you're right, the narrative makes more sense than the world description in terms of the Wanderers campaign. That should probably get changed.
I could change these lines to only show up after the Unfettered invasion, and edit them to imply that their military was only very recently put together.

Reply:
That would definitely be workable, and flavourful.

Reply:
I tried editing some lines and adding others, is it better now?

Reply:
I would lose the conscripted one, but otherwise I'd say these are excellent.

Reply:
The `"I was [conscripted, forced] into service in a warship...` one?

Reply:
Yeah that one, the wanderers seem perfectly willing to pull together in a crisis. Conscription shouldn't be necessary.